### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by MukundVarmaT** [warner/magic-wormhole#393]
Probably this is just me but wouldn't it be great if 

```bash
Sending X GB file named '<unnamed>.zip'
On the other computer, please run: wormhole receive
Wormhole code is: blah-blah-blah
```

becomes 

```bash
Sending X GB file named '<unnamed>.zip'
On the other computer, please run: wormhole receive blah-blah-blah
```

so that we can just copy-paste the wormhole receive command and paste it on the remote server without the hassle of typing wormhole receive and then copy-pasting the code or even typing the whole thing.  

Thank you!

**Comment by warner** [comment]
Good idea. For anyone inclined to take this on, the files to change are:

* [cli/cmd_send.py line 129](https://github.com/magic-wormhole/magic-wormhole/blob/d517a10282d5e56f300db462b1a6eec517202af7/src/wormhole/cli/cmd_send.py#L129) : change the `print` command to use the new message (the code is in `code`, but pay attention to the way we build up `other_cmd` because sometimes it includes `--verify`)
* [test/test_cli.py](https://github.com/magic-wormhole/magic-wormhole/blob/master/src/wormhole/test/test_cli.py): there are 6 places where the unit tests check the output of the command which need to be updated

**Comment by MukundVarmaT** [comment]
Hey @warner, I thought I could try patching this. However, what I noticed is that the message is different when run from a local machine vs when run from a cloud server and I am not sure why.

When run from a local machine:
```
Sending X GB file named '<unnamed>.zip'
Wormhole code is: blah-blah-blah
On the other computer, please run:

wormhole receive blah-blah-blah
```

When run from a remote server
```
Sending X GB file named '<unnamed>.zip'
On the other computer, please run: wormhole receive
Wormhole code is: blah-blah-blah
```

And as per the code [cli/cmd_send.py line 125](https://github.com/magic-wormhole/magic-wormhole/blob/d517a10282d5e56f300db462b1a6eec517202af7/src/wormhole/cli/cmd_send.py#L125), it is supposed to print the wormhole code first followed by the command to run in [line 129](https://github.com/magic-wormhole/magic-wormhole/blob/d517a10282d5e56f300db462b1a6eec517202af7/src/wormhole/cli/cmd_send.py#L129)

**Comment by warner** [comment]
@MukundVarmaT were maybe those two computers running different versions of the `wormhole` executable? I think we might have incorporated part of this change already.

Incidentally, while cut-and-paste of the sender-recommended command *is* a common way to use it (e.g. if you're `ssh`ed into a remote computer and you're sending a file to yourself on the other machine), another very common approach is for two people to speak the wormhole code to each other. I've lost count of how many times I've sent a file to a coworker across the room, and I just yell "I'm wormholeing it to you, the code is 4-purple-sausages". I have to explain how to run `wormhole receive` the first time, but after that I don't need to tell them what to type.

And, if you're doing it that way, it's often *easier* to type `wormhole receive` (or just `wormhole rx` which works because I'm a lousy speller :-) and then interactively type the wormhole code into the receiver, because then you get tab completion on the words. If you type the entire code into the command ahead of time (`wormhole receive 4-purple-sausages`), then you don't get to benefit from the completion. Instead, I do `wormhole rx` and then `4-p TAB` and then `sa TAB` and get to the right words very quickly.

So, however we change this (assuming we haven't changed it adequately enough already), I'd like to not make it look like the only way to use it is to type the full code in ahead of time.
